### DISCUSSION TITLE
Issues with ProRender 3.2 / Blender 3.0.0 on Mojave

### DISCUSSION THREAD

**Discussion by Jamster007** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#446] [Q&A]
Hi there.  I can't get this to work on Mojave with my RX580.  Looking through the original release notes, only legacy mode seems to be supported on Mojave - where do I set the render quality mode to Legacy?  Thanks

**Comment by ARozghon** [comment]
Hi, thank you for your request. Now legacy mode deleted from UI, and now RPR for Blender used Full, Interactive (for RT GPUs), and Low/Med/High.
